config/has_128bit_floatmax_t.cpp

```diff
@@ -4,13 +4,13 @@
 //  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 #include <boost/cstdfloat.hpp>
-#include <boost/static_assert.hpp>
+#include <boost/math/tools/assert.hpp>
 
 #ifndef BOOST_FLOAT128_C
 #error "There is no 128 bit floating point type"
 #endif
 
-BOOST_STATIC_ASSERT(sizeof(boost::floatmax_t) * CHAR_BIT == 128);
+BOOST_MATH_STATIC_ASSERT(sizeof(boost::floatmax_t) * CHAR_BIT == 128);
```

Comment:
Is there any reason not to just use `static_assert` here?

Reply:
The message in `static_assert` did not become optional until C++17.

Reply:
Yes I know, we would have to invent some descriptive error messages.

Reply:
That's fair.